docs/concepts/promptflow-vs-langchain.md

```diff
@@ -0,0 +1,49 @@
+# Why build prompt flow when there is LangChain?
+
+When we started this project, [LangChain](https://www.langchain.com/) already became popular esp. after the ChatGPT launch. One of the questions we’ve been asked is what’s the difference between prompt flow and LangChain. This article is to elucidate the reasons for building Prompt Flow and the deliberate design choices we have made. To put it succinctly, prompt flow is a suite of development tools for you to build LLM apps with a strong emphasis of quality, not a framework - which LangChain is.
+
+While LLM apps are mostly in exploration stage, Microsoft started in this area a bit earlier and we’ve had the opportunity to observe how developers are integrating LLMs into existing systems or build new applications. These invaluable insights have shaped the fundamental design principles of Prompt Flow. 
+
+## 1. Highlighting the Significance of Prompts
```

Comment:
Expose the prompts vs. hiding them

- might be a better framing from what you described below :D